src/pseudopeople/noise.py

```diff
@@ -16,12 +16,12 @@
 
 from pseudopeople.entity_types import ColumnNoiseType, RowNoiseType
 from pseudopeople.noise_entities import NOISE_TYPES
-from pseudopeople.schema_entities import Form
+from pseudopeople.schema_entities import FORMS
 from pseudopeople.utilities import get_randomness_stream
 
 
 def noise_form(
-    form: Form,
+    form: FORMS,
```

Comment:
FORMS is an instance of the __Forms class, ie the form typing is `form: __Forms`

Reply:
Actually the correct typing is `form: pseudopeople.schema_entities.Form` since you're passing a single Form, not all forms.